### PR TITLE
Changes to application checklist form

### PR DESCRIPTION
**Pull request by stemount**
Improvements to the validation of this form and also changing visible states for partnership application checklist form.

### CODE REVIEW

File: web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsApplicationAuthorityChecklistForm.php [hunk 1]
```diff
@@ -202,18 +205,17 @@ public function validateForm(array &$form, FormStateInterface $form_state) {
...
-      elseif ($form_state->getValue('business_regulated_by_one_authority') &&
-        !$form_state->getValue('is_local_authority') &&
-        !$form_state->getValue('business_informed_local_authority_still_regulates')) {
-        $this->setElementError(['section_two','business_informed_local_authority_still_regulates'], $form_state, 'The business needs to be informed about local authority.');
 
+      // Warn that business needs to be informed their local authority still regulates.
+      if ($form_state->getValue('business_regulated_by_one_authority') == 1 &&
+        $form_state->getValue('is_local_authority') == 0 &&
+        $form_state->getValue('business_informed_local_authority_still_regulates') == 0) {
+        $this->setElementError(['section_two','business_informed_local_authority_still_regulates'], $form_state, 'The business needs to be informed about local authority.');
```
Comment (jon-edery) [inline web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsApplicationAuthorityChecklistForm.php]: "about local authority." => about **_the_** local authority.

Comment (stemount) [reply web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsApplicationAuthorityChecklistForm.php]: @jon-edery aye this is what the ticket says… what you think @nao360 ? 👎

Comment (jon-edery) [reply web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsApplicationAuthorityChecklistForm.php]: In that case we can just merge if needed we can update later.

**Comment by stemount** [comment]
cheers @jon-edery

**Pull request merged**